docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -0,0 +1,220 @@
+[[modules-remote-clusters]]
+== Remote clusters
+
+The _remote clusters_ module allows establishing uni-directional connections to
+a remote cluster.
+
+Remote cluster connections work by configuring a remote cluster and connecting
+only to a limited number of nodes in the remote cluster. Each remote cluster is
+referenced by a name and a list of seed nodes.  When a remote cluster is
+registered, its cluster state is retrieved from one of the seed nodes so that up
+to 3 _gateway nodes_ are selected to be connected to as part of remote cluster
+requests.  Remote cluster connections consist of uni-directional connections
+from the coordinating node to the previously selected remote nodes only. It is
+possible to tag which nodes should be selected through node attributes (see
+<<remote-cluster-settings>>).
+
+Each node in a cluster that has remote clusters configured connects to one or
+more _gateway nodes_ and uses them to federate requests to the remote cluster.
+
+ifndef::include-xpack[]
+This functionality is used in
+<<modules-cross-cluster-search,cross-cluster search>>, and possibly other
```

Comment:
I think it'd be nice to move this up to the first paragraph if you can. Maybe not worded exactly like this, but similar. I'd also drop ", and possible other features." If we want to say that it might be used for more things in the future then we should say that. Otherwise I'd just leave that out entirely.